packages/playground/src/playground-zip.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+defined('ABSPATH') || exit;
+
+require __DIR__ . '/playground-db.php';
+
+use ZipStream\ZipStream;
+use ZipStream\Option\Archive;
+
+function playground_zip_wp_content($zip)
+{
+	$root_dir = WP_CONTENT_DIR;
+	$directory = new \RecursiveDirectoryIterator($root_dir, \FilesystemIterator::FOLLOW_SYMLINKS);
+	$iterator = new \RecursiveIteratorIterator($directory);
+	$regex = new \RegexIterator($iterator, '/^.+\/(?!.*\.\.).+$/i', \RecursiveRegexIterator::GET_MATCH);
```

Comment:
Let's document what that regex is doing, I can't tell without stopping here for a few minutes

Reply:
Done in  e0cb7c58fe6785d24fbc53006677c1d8f5667f76

Reply:
Aha, thank you! This will exclude `.htaccess` files, the `.ht.sqlite` database file, and other potentially desirable files. I'm not sure what's the right thing to do here, but we could start by allowing those two.

Reply:
`.htaccess` isn't and a `@TODO` note would be good enough for now. The `.ht.sqlite` file would normally be used if available, but in this case we're moving the data as SQL dump so I guess that isn't used either. A comment or an issue should do the trick.

Reply:
Done in c02934d36a2fe2dec4ad6d871179182c3a8f03e6